src/init.cpp

```diff
@@ -558,7 +558,7 @@ bool AppInit2()
     
     fUseFastIndex = GetBoolArg("-fastindex", false);
 
-    nMinerSleep = GetArg("-minersleep", 500);
+    nMinerSleep = GetArg("-minersleep", 8000);
```

Comment:
Actually, thinking about this again... There's a very specific second where staking is allowed, given the staking mask, right? With the previous 500ms sleep you were pretty much guaranteed to hit that window but with the new sleep there's a risk that you always sleep when the staking window is open. Or did I misinterpret the purpose of the mask?